### PR TITLE
Read ipv6 label from node and send it via eventforwarder

### PR DESCRIPTION
**Pull request by lftakakura**

### CODE REVIEW

File: models/room.go [hunk 1]
```diff
@@ -30,6 +30,9 @@ const (
 	RoomTerminating = "terminating"
 	// RoomTerminated string representation
 	RoomTerminated = "terminated"
+
+	// Ipv6LabelKey is the label key on kubernetes
+	Ipv6LabelKey = "tfg.io/ipv6"
```
Comment (Henrod) [inline models/room.go]: You should not use tfg as key. This prefix should be gotten from config file

File: models/room.go [hunk 2]
```diff
@@ -42,8 +45,9 @@ type Room struct {
 
 // RoomAddresses struct
 type RoomAddresses struct {
-	Ports []*RoomPort `json:"ports"`
-	Host  string      `json:"host"`
+	Ports     []*RoomPort `json:"ports"`
+	Host      string      `json:"host"`
+	Ipv6Label string      `json:"Ipv6Label"`
```
Comment (Henrod) [inline models/room.go]: `json:"ipv6Label"`

**Comment by Henrod** [comment]
Good. But the testes are failing

**Comment by lftakakura** [comment]
> Good. But the testes are failing

I know. Should be ok now!

**Pull request merged**